test/test_acceptance_tests.py

```diff
@@ -0,0 +1,172 @@
+import io
+import sys
+import unittest
+from unittest.mock import patch
+from src.game import Game
+from test.mocks.mock_message import MockMessage
+
+
+class TestAcceptance(unittest.TestCase):
+    @patch("builtins.input", side_effect=["1", "1", "2", "3", "4", "5", "6", "7", "n"])
+    def test_player_X_wins_the_game(self, mock_input):
+        captured_output = io.StringIO()
+        game = Game(message=MockMessage())
+        expected_message = MockMessage.declare_winner(self, "X")
+
+        sys.stdout = captured_output
+        game.run()
+        sys.stdout = sys.__stdout__
+
+        self.assertIn(expected_message, captured_output.getvalue())
+
+    @patch("builtins.input", side_effect=["1", "1", "2", "3", "5", "4", "8", "n"])
+    def test_player_O_wins_the_game(self, mock_input):
+        captured_output = io.StringIO()
+        game = Game(message=MockMessage())
+        expected_message = MockMessage.declare_winner(self, "O")
+
+        sys.stdout = captured_output
+        game.run()
+        sys.stdout = sys.__stdout__
+
+        self.assertIn(expected_message, captured_output.getvalue())
+
+    @patch(
+        "builtins.input",
+        side_effect=["1", "1", "3", "2", "4", "5", "9", "6", "8", "7", "n"],
+    )
+    def test_play_through_ends_in_draw(self, mock_input):
+        captured_output = io.StringIO()
+        game = Game(message=MockMessage())
+        expected_message = MockMessage.game_over_message(self)
+
+        sys.stdout = captured_output
+        game.run()
+        sys.stdout = sys.__stdout__
+
+        self.assertIn(expected_message, captured_output.getvalue())
+
+    @patch(
+        "builtins.input",
+        side_effect=[
+            "1",
+            "1",
+            "3",
+            "2",
+            "4",
+            "5",
+            "9",
+            "6",
+            "8",
+            "7",
+            "y",
+            "1",
+            "1",
+            "2",
+            "3",
+            "5",
+            "4",
+            "8",
+            "n",
+        ],
+    )
```

Comment:
there is no way to override how this is linting?  .. when linters work against you.

Reply:
Mmmm...I could check and see but I'm not sure...

Reply:
don't let it block you; but sometimes, the linters get it wrong.  I think this is one of those times.